web/app.php

```diff
@@ -1,14 +1,18 @@
 <?php
 
-use Symfony\Component\ClassLoader\ApcClassLoader;
 use Symfony\Component\HttpFoundation\Request;
 
-$loader = require_once __DIR__.'/../app/bootstrap.php.cache';
+require __DIR__.'/../vendor/autoload.php';
 
-require_once __DIR__.'/../app/AppKernel.php';
+if (PHP_VERSION_ID < 70000) {
```

Comment:
Since we only support 7.0 or higher, these if statements can be removed

Reply:
Good point, updated this!